buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -95,6 +95,127 @@ class AssemblyResourceGeneration extends DefaultTask {
     public static final String GRANDFATHERED_TYPES = [VANTIQ_DOCUMENTS, VANTIQ_PROCEDURES, VANTIQ_PROJECTS,
                                                       VANTIQ_RULES, VANTIQ_SOURCE_IMPLEMENTATION, VANTIQ_TYPES]
 
```

Comment:
Set of type names & values as used by the UI.  This information was missing, and that caused issues after a CLI-based import.